daemon/daemon_unix.go

```diff
@@ -471,7 +471,7 @@ func verifyPlatformContainerResources(resources *containertypes.Resources, sysIn
 		resources.KernelMemory = 0
 	}
 	if resources.KernelMemory > 0 && resources.KernelMemory < linuxMinMemory {
-		return warnings, fmt.Errorf("Minimum kernel memory limit allowed is 4MB")
+		return warnings, fmt.Errorf("Minimum kernel memory limit allowed is 6MB")
```

Comment:
Should this instead use our "data size humanizer" functions directly on `linuxMinMemory` so it doesn't mismatch again? (Maybe not worth it?)